lib/xlibc/src/threads/thrd_current.c

```diff
@@ -0,0 +1,8 @@
+#include <threads.h>
+#include "xboxkrnl/xboxkrnl.h"
+
+thrd_t thrd_current (void)
+{
+    // This is just a pseudohandle (enough for thrd_equal)
+    return (thrd_t) 0xFFFFFFFE;
```

Comment:
Why not `KeGetCurrentThread` or something otherwise unique?

Reply:
That would only solve the problem (`thrd_current()` only being useful for `thrd_equal()`) if I could use the KTHREAD pointer for everything else as well (like `thrd_join()`) or somehow use it to get the original handle (although that's probably not how handles are meant to be used). Also, I'd have to somehow get the KTHREAD pointer upon thread creation.
I thought about using the thread ID (a.k.a. `UniqueThread`, which you can get from `PsCreateSystemThreadEx()`), but that suffers the same problem.